pkg/kubelet/cm/deviceplugin/manager.go

```diff
@@ -580,16 +562,16 @@ func (m *ManagerImpl) allocateContainerResources(pod *v1.Pod, container *v1.Cont
 		metrics.DevicePluginAllocationLatency.WithLabelValues(resource).Observe(metrics.SinceInMicroseconds(startRPCTime))
 		if err != nil {
 			// In case of allocation failure, we want to restore m.allocatedDevices
-			// to the actual allocated state from m.podDevices.
+			// to the actual allocated state from m.allodation.
 			m.mutex.Lock()
-			m.allocatedDevices = m.podDevices.devices()
+			m.allocatedDevices = m.allocation.devices()
 			m.mutex.Unlock()
 			return err
 		}
 
-		// Update internal cached podDevices state.
+		// Update internal cached allocation state.
 		m.mutex.Lock()
-		m.podDevices.insert(podUID, contName, resource, allocDevices, resp)
+		m.allocation.insert(podUID, contName, resource, allocDevices, resp)
```

Comment:
Should we rename `allocDevices` to `allocResources` and likewise at other places as well? `Devices` is there for device files in `AllocateResponse` and that makes it confusing.